go.graphics.swing/src/main/java/go/graphics/swing/text/JOGLTextDrawer.java

```diff
@@ -19,32 +19,43 @@
 import go.graphics.text.EFontSize;
 import go.graphics.text.TextDrawer;
 
-import java.awt.Font;
+import java.awt.*;
 import java.awt.geom.Rectangle2D;
 
 /**
  * This class is a text drawer used to wrap the text renderer.
- * 
+ *
  * @author michael
  */
 public final class JOGLTextDrawer implements TextDrawer {
 
 	private static final String FONTNAME = "Arial";
 
+	private static final int DEFAULT_DPI_WIDTH = 1920;
+	private static float SCALING_FACTOR = 1;
+
+	public static void calculateScalingFactor() {
```

Comment:
should we just use a static{} block here. Then this will be calculated when the class if first used. I don't like having to call this method from SwingManagedJSettlers.